ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html

```diff
@@ -1,29 +1,33 @@
+
+        <image src="/assets/images/moj-logotype-crest.png" xlink:href="" class="moj-header__logotype-crest-fallback-image" width="40" height="40"></image>
+
+      </svg>
+
+      <a class="moj-header__link moj-header__link--organisation-name" [routerLink]="">HM Courts &amp; Tribunals Service</a>
+
     </div>
+    <div class="moj-header__content">
```

Comment:
The __  is this standard recall a previous comment suggesting it should be - are you bale to confirm?

Reply:
yea, thats the style based on the documentation:
https://design-patterns.service.justice.gov.uk/components/header/